syncd/Syncd.cpp

```diff
@@ -3170,6 +3170,8 @@ void Syncd::sendShutdownRequest(
         return;
     }
 
+    s = sai_serialize_switch_shutdown_request(switchVid);
```

Comment:
This looks good. 
Can you please move the check, if (m_notifications == nullptr) to the start of the function, so we log the line, "sending switch_shutdown..." only when we could send.